server/src/test/java/org/apache/druid/client/BrokerSegmentWatcherConfigTest.java

```diff
@@ -45,9 +45,10 @@ public void testSerde() throws Exception
     );
 
     Assert.assertNull(config.getWatchedTiers());
+    Assert.assertNull(config.getIgnoredTiers());
 
     //non-defaults
-    json = "{ \"watchedTiers\": [\"t1\", \"t2\"], \"watchedDataSources\": [\"ds1\", \"ds2\"] }";
+    json = "{ \"watchedTiers\": [\"t1\", \"t2\"], \"watchedDataSources\": [\"ds1\", \"ds2\"], \"ignoredTiers\": [\"t3\", \"t4\"] }";
```

Comment:
this test is sort of strange now that it is an invalid configuration, though technically it isn't enforced by the config itself so it isn't problematic here. maybe we should at least leave a comment that this config is illegal in practice?

Reply:
No, you are right, we should not use an invalid configuration here. Fixing.